packages/paste-core/components/toast/src/ToastPortal.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {StyledBase} from '@twilio-paste/theme';
+import {Portal} from '@twilio-paste/reakit-library';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+export interface ToastPortalProps {
+  children: NonNullable<React.ReactNode>;
+}
+
+const ToastPortal: React.FC<ToastPortalProps> = ({children, ...props}) => {
+  return (
+    <Portal>
+      {/* import Paste Theme Based Styles due to portal positioning. */}
+      <StyledBase>
+        <Box {...safelySpreadBoxProps(props)} position="fixed" right="space40" top="space40" zIndex="zIndex90">
```

Comment:
The position to the window is shared across `ToastContainer` and `Toaster`. So both the `Toaster` and `ToastContainer` now use the `ToastPortal` for positioning. 

First approach was to try and use the `ToastContainer` for the `Toaster`, but as `ToastContainer` came first, I couldn't really think of a nice way to not introduce a breaking change to anyone already using it as a basic toast positioner because of the use of `Stack`. So all I did was abstract the common things; needs to be a portal, needs base styles and positioned to the window top right. I don't _love_ it, but hoping the `ToastContainer` is deprecated in the future.